tests/functional/test_api.py

```diff
@@ -87,6 +101,34 @@ def user_with_token(user, db_session, factories):
 
 
 @pytest.fixture
+def auth_client(db_session, factories):
+    auth_client = factories.AuthClient(authority='thirdparty.example.org')
+    db_session.commit()
+    return auth_client
+
+
+@pytest.fixture
+def third_party_user(auth_client, db_session, factories):
+    user = factories.User(authority=auth_client.authority)
+    db_session.commit()
+    return user
+
+
+@pytest.fixture
+def publisher_group(auth_client, db_session, factories):
+    group = factories.PublisherGroup(authority=auth_client.authority)
```

Comment:
The `PublisherGroup` factory inherits from the `Group` factory, so I think it’s picking up the [`creator` definition from there](https://github.com/hypothesis/h/pull/4333/files#diff-149b85565f69bed7e895157dfc37e09aL103).